### PR TITLE
Patch 7

### PR DESCRIPTION
**Pull request by daltskin**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@daltskin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@daltskin Can you check the deployment error? https://travis-ci.org/Azure/azure-quickstart-templates/builds/134542610#L230

**Comment by daltskin** [comment]
Failure is because the VM image is not found.  These images are available as part of an Azure MSDN sub.

File: visual-studio-dev-vm-chocolatey/azuredeploy.json [hunk 1]
```diff
@@ -95,7 +86,7 @@
     },
     "setupChocolatelyScriptLocation": {
       "type": "string",
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/visual-studio-dev-vm-chocolatey/",
+      "defaultValue": "https://raw.githubusercontent.com/daltskin/ARMChocolatey/master/",
```
Comment (singhkay) [inline visual-studio-dev-vm-chocolatey/azuredeploy.json]: The script should be placed in alongside the template

**Comment by singhkay** [comment]
@daltskin Got it. Can you look into placing the script alongside the template? Also see our folder organization guide https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/images/namingConvention.png

**Comment by daltskin** [comment]
CI Failure is because the VM image is not found. These images are available as part of an Azure MSDN sub.

**Comment by singhkay** [comment]
@daltskin np for that, merged.

**Pull request merged**